woocommerce/google-listings-and-ads
# Remove product feed coming soon notice and show reports by default.

### Changes proposed in this Pull Request:

Removes the coming soon notice added the product feed page in https://github.com/woocommerce/google-listings-and-ads/pull/533 as the product feed is now functional / enabled in `trunk`.

Similarly, this PR sets reports to show by default now that both program and product reports are retrieving data from Google. I've left the filter in the code on the off chance we need to flip things around again or provide a workaround for some merchants to remove the reports.

Closes #628

### Detailed test instructions:

1. Clone, install and build
2. Remove the `gla_enable_reports` filter in custom code if you are using it to show reports
3. Go to the GLA dashboard and check that "View Reports" button is visible and "Reports" tab is visible
4. Browse to Reports > Programs & Reports > Products - confirm they load as expected for you (whether you have mocked data or otherwise)
3. Browse to the Product Feed page. Confirm that the coming soon notice is not visible. Check that products are listed in the product feed table.

### Changelog Note:

> Tweak - Remove product feed coming soon notice and show reports by default.
